schemas/contract-state.json

```diff
@@ -0,0 +1,65 @@
+{
```

Comment:
you're right - this could use an update, but i'd rather do it in a separate PR and have EVERY test run an AJV validation against the resulting schema.

Reply:
Fair enough - i didnt change it at all, just moved it from the top level to the schemas folder and renamed it from "json_schema" to "contract-state", we should definitly have a ticket to update this. Noticed some issues in the tooling (backfilled with ms instead of seconds) that a state validator from this would fix.